packages/datadog-instrumentations/src/helpers/hook.js

```diff
@@ -20,7 +20,8 @@ function Hook (modules, onrequire) {
     const parts = [moduleBaseDir, moduleName].filter(v => v)
     const filename = path.join(...parts)
 
-    if (this._patched[filename]) return moduleExports
+    // this does not seem to work with IITM: the same file seems to import different stuff
+    // if (this._patched[filename]) return moduleExports
```

Comment:
I can't specify file in the vitest's `addHook` calls because they're hashed, and this seems to interfere. TODO: check this more thoroughly